### DISCUSSION TITLE
derivatives and webhooks unavailable

### DISCUSSION THREAD

**Discussion by ignorus** [AntlerVC/firetable#1332] [Support / Q&A] [answered]
I synced my data from firestore, updated my billing on GCP, deployed the rowy on cloud run services

```
rowy-backend | 0 | southamerica-east1 | Allow unauthenticated | All |   | 1 hour ago | |  
rowy-hooks | 0 | southamerica-east1 | Allow unauthenticated | All |   | 1 hour ago |  |  
```

but whenever I try to build a function/webhook, I says, "U need to configure", so I click continue and it says "alredy configured". But  it never lets me build my function/webhook

I'm using google chrome BTW

**Comment by BeeBombshell** [comment, marked as answer]
Hey @ignorus, could you try following the steps as described here: https://github.com/rowyio/rowy/discussions/1087#discussioncomment-4756550 ?
